self-tests.yml

```diff
@@ -829,7 +831,6 @@ buildvariants:
       - name: test-cloud
       - name: "js-test"
       - name: generate-api-docs
-      - name: push-api-docs
```

Comment:
How come you removed this?

Reply:
There's no task with this name! I think we must have merged them in some point.